include/seqan3/alignment/configuration/align_config_result.hpp

```diff
@@ -23,26 +23,42 @@ namespace seqan3::detail
  * \ingroup alignment_configuration
  */
 struct with_score_type
-{};
+{
+    //!\privatesection
+    //!\brief The internal rank for computing a score.
```

Comment:
Maybe the desctiption is a little odd. I will reformulate it. But just that you can better understand the rational. Before this change, these types were distinct, i.e. I couldn't subsume the configurations. So I had to check, do you want to compute the score? Then only compute the score. Do you want to compute the alignment? Ok, then compute the score, the coordinates and the alignment. Now I can say? Do you want to compute the score? Ok, compute it. DO you want to also compute the alignment? Good then also compute the alignment but note, that the score is already computed because computing the alignment means computing the score.